bot/university/university.py

```diff
@@ -0,0 +1,378 @@
+            persons = xml.findall("person")
+            index = 0
+
+            if len(persons) > 0:
+                if len(persons) > 1:
+                    index = await self._staff_selection(ctx.author, ctx.channel, persons)
+
+                response = requests.get(constants.URL_UFIND_API + "/staff/" + persons[index].attrib["id"])
+                response.encoding = "utf-8"
+
+                staff_data = _parse_staff_xml(response.text)
+                embed = _create_embed_staff(staff_data)
+                await ctx.channel.send(embed=embed)
+            else:
+                raise ValueError("No person with the specified name was found.")
+        else:
+            response.raise_for_status()
+
+    @ufind_get_staff_data.error
+    async def ufind_error(self, ctx, error: commands.CommandError):
+        """Error Handler for the `ufind` subcommand `staff`.
+
+        Handles specific exceptions which occur during the execution of this command. Other exceptions will be simply
+        re-raised so that they can be handled somewhere else.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            error (commands.CommandError): The error raised during the execution of the command.
+        """
+        if isinstance(error.original, ValueError):
+            await ctx.send("Ich konnte leider niemanden unter dem von dir angegeben Namen finden. :slight_frown: "
+                           "Möglicherweise hast du dich verschrieben.")
+        else:
+            raise error
+
+    async def _staff_selection(self, author: discord.User, channel: discord.TextChannel, persons: List[ET.Element]) \
+            -> int:
+        """Method for handling multiple results when searching for a staff member on `ufind`.
+
+        If multiple results have been returned by `ufind`, this method will post an embed containing the names of these.
+        The user can then choose one person by reacting to the embed. If the user takes too long and the timeout is
+        reached, the embed will be removed and the bot will post a message to inform the user about what happened.
+
+        Args:
+            author (discord.User): The user who invoked the command.
+            channel (discord.TextChannel): The channel in which the command has been invoked.
+            persons (List[ET.Element]): A list containing the person elements from the XML string returned by `ufind`.
+
+        Returns:
+            int: The index of the selected person.
+        """
+        timeout = 15.0
+        list_selection_emojis = SelectionEmoji.to_list()
+
+        embed = _create_embed_staff_selection(persons)
+        message = await channel.send(embed=embed)
+
+        for i in range(len(embed.fields)):
+            await message.add_reaction(list_selection_emojis[i])
+
+        def check(_reaction, user):
+            return user == author and SelectionEmoji(_reaction.emoji) is not None
```

Comment:
You could write this as lambda, does not change anything, just looks more concise. 

```
check = lambda _reaction, user : user==author and SelectionEmoji(_reaction.emoji) is not None
```

Allows to omit the return.

Reply:
Even though I like lambdas, we gain no benefit in using one here. PEP 8 states the following:

> Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.

And even if I don't use an assignment here, it only makes the code less readable.